tests/delegated_scanning_test.go

```diff
@@ -614,6 +614,40 @@ func (ts *DelegatedScanningSuite) TestAdHocScans() {
 	})
 }
 
+// TestAdHocCentralScan tests ad-hoc image scan requests without specifying a cluster.
+func (ts *DelegatedScanningSuite) TestAdHocCentralScan() {
+	ctx := ts.ctx
+
+	// Helper to build a scan request without specifying a cluster.
+	scanImgReq := func(imageStr string) *v1.ScanImageRequest {
+		return &v1.ScanImageRequest{
+			ImageName: imageStr,
+			Force:     true,
+			Cluster:   "", // Explicitly no cluster specified
+		}
+	}
+
+	ts.Run("scan image with minimal config and no cluster specified", func() {
+		ts.setConfig(ctx, ts.ubi9Image)
+
+		ps := &storage.PermissionSet{
+			Name: deleScanPermissionSetName,
+			ResourceToAccess: map[string]storage.Access{
+				"Image": storage.Access_READ_WRITE_ACCESS,
+			},
+		}
+
+		role := &storage.Role{
+			Name:          deleScanRoleName,
+			AccessScopeId: allowAllAccessScope,
+		}
+
+		conn := ts.getLimitedCentralConn(ctx, ps, role)
```

Comment:
This test doesn't need to test permissions, suggest removing this and using the default `conn` in `TestAdHocScans`